bitbots_utils/scripts/tf_map_odom.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+import rclpy
+from rclpy.node import Node
+from rclpy.duration import Duration
+
+from geometry_msgs.msg import TransformStamped
+from gazebo_msgs.msg import ModelStates
+import tf2_ros as tf2
+
+
+
+class TFWorld(object):
+    def __init__(self):
+        rclpy.init()
+        node = Node("map")
+        br = tf2.TransformBroadcaster(node)
```

Comment:
Afaik this could be a StaticTransformBroadcaster which does not need a loop.

Reply:
Good point, I just transferred it to ROS2 without thinking about it. I will change it